complex/complex.cr

```diff
@@ -0,0 +1,11 @@
+require "colorize"
+require "complex"
+x1 = Complex.new(1, 0)
+x2 = Complex.new(2, 1)
+pp x1
+pp x2
+puts
+p "x 1 + x2 = #{x1 + x2}"
```

Comment:
`x 1` -> `x1`

But the the better way would be `pp x1 + x2` which prints almost the same:

```crystal
> x1, x2 = 1, 2

> p "x1 + x2 = #{x1 + x2}"
"x1 + x2 = 3"

> pp x1 + x2
x1 + x2 # => 3
```